sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/AbstractScheduler.java

```diff
@@ -236,32 +235,20 @@ public void registered(SchedulerDriver driver, Protos.FrameworkID frameworkId, P
                 return;
             }
 
+            Driver.setDriver(driver);
+
             LOGGER.info("Registered framework with frameworkId: {}", frameworkId.getValue());
-            this.driver = driver;
             this.reviveManager = new ReviveManager(driver);
```

Comment:
Is there a reason that the `ReviveManager` doesn't use the new singleton driver?

Reply:
No, it's an oversight.  I'm guess I missed a couple.